combat-trainer.lic

```diff
@@ -1146,6 +1146,10 @@ class AbilityProcess
 
     @kneel_khri = settings.kneel_khri
     echo("  @kneel_khri: #{@kneel_khri}") if $debug_mode_ct
+
+    @barb_buffs = @buffs.delete('barb_buffs') || []
+    echo("  @barb_buffs: #{@barb_buffs}") if $debug_mode_ct
+    setup_barb_buff_flags if @barb_buffs.any?
```

Comment:
What would be the use of `any?` then? It's used here `@wounds.any?`, etc. This is just for my own understanding.